Moli/Core/Screens/Home/Live/HostItemView.swift

```diff
@@ -12,11 +14,27 @@ struct HostItemView: View {
         return host.state.liveThumbnailUrl ?? host.profile.thumbnails?.M.url
     }
     
+    private func getFrameHeight() -> CGFloat {
+        if numberOfHosts <= 2 {
+            return 200
+        } else if numberOfHosts == 3 && index == 2 {
+            return 200
+        } else {
+            print(index)
+            return 96
+        }
+    }
+    
     var body: some View {
         VStack(alignment: .leading, spacing: 0) {
             if let imageURL = getImageURL(), let url = URL(string: imageURL) {
                 AsyncImage(url: url) { image in
-                    image.resizable()
+                    image
+                        .resizable()
+                        .scaledToFill()
+                        .frame(height: getFrameHeight())
+                        .frame(minWidth: 0, maxWidth: .infinity)
```

Comment:
:art: Tror du får samma beteende av att bara ha maxWidth
```suggestion
                        .frame(maxWidth: .infinity)
```